question16/main.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+const createElementWithClassName = (element, name) => {
+  const createdElement = document.createElement(element);
+  createdElement.classList.add(name);
+  return createdElement;
+};
+
+const newsWrapper = document.getElementById('js-newsWrapper');
+const tabsGroup = document.getElementById('js-tabs');
+const contentsWrapper = document.createElement("div");
+contentsWrapper.id = "js-contentsWrapper";
+contentsWrapper.classList.add("contentsWrapper");
+
+
+//Get json data
+async function callApi() {
+  try {
+    const res = await fetch('https://myjson.dit.upm.es/api/bins/an1r');
+    const json = await res.json();
+    return json.data;
+  }
+  catch(error) {
+    tabsGroup.textContent = "データが取得できませんでした";
+    console.error(error);
+  }
+  finally {
+    console.log('CONPLETE!');
+  }
+}
+
+async function init() {
+  const result = await callApi();
+  if (result){
+    renderElement(result);
+    } else {
+    tabsGroup.textContent = "データの表示に失敗しました";
```

Comment:
I think this part is the same,
it's not a list, so it seems better to change the tag.